intercom/params.go

```diff
@@ -0,0 +1,61 @@
+package intercom
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/paramsbuilder"
+	"golang.org/x/oauth2"
+)
+
+const (
+	// DefaultPageSize is number of elements per page.
+	DefaultPageSize = 50
```

Comment:
However, the docs say it is 150. In practice some endpoints have a limit of 60. Exceeding it returns bad request instead of defaulting to their limit :man_shrugging: .

Reply:
Why 50 instead of 60 then?

Reply:
50 is large enough and under 60 to be safe. Also this number looks better.
The arguments on my side are not solid at all :sweat_smile:

Reply:
Changes: https://github.com/amp-labs/connectors/pull/346/commits/9a19a0c4e50cd7829cc1622e64dc61bcd1033041